src/processor/Package.ts

```diff
@@ -83,4 +86,30 @@ export class Package {
       pullAt(sd.rules, rulesToRemove);
     });
   }
+
+  private constructNamedExtensionContainsRules(): void {
+    [...this.profiles, ...this.extensions].forEach(sd => {
+      const rulesToRemove: number[] = [];
+      sd.rules.forEach(rule => {
+        if (rule instanceof ExportableContainsRule && rule.path.endsWith('extension')) {
+          rule.items.forEach(item => {
+            const onlyRuleIdx = sd.rules.findIndex(
+              other =>
+                other.path === `${rule.path}[${item.name}]` && other instanceof ExportableOnlyRule
+            );
+            const onlyRule = sd.rules[onlyRuleIdx] as ExportableOnlyRule;
+            // Explicitly ignore "Extension" since some IGs (ex: USCore) add a type constraint to Extension
+            // on the differential unnecessarily. Using the "named" syntax with "Extension" causes errors in SUSHI.
+            // As long as the type is not "Extension", we assume it is a profile of Extension, and we can therefore
+            // use the "named" syntax.
+            if (onlyRule && onlyRule.types.length === 1 && onlyRule.types[0].type !== 'Extension') {
```

Comment:
Very minor, but it should be safe to rewrite this statement with some nullish coalescing operators:
```suggestion
            if (onlyRule?.types.length === 1 && onlyRule?.types[0].type !== 'Extension') {
```

It should be safe, since if `onlyRule` is undefined, the first condition will be false.